test/e2e/integration/cloud_backup_test.go

```diff
@@ -37,13 +37,14 @@ import (
 func startEtcd() (*Cmd, *chan error) {
 	errChan := make(chan error)
 	logger := logrus.New()
-	etcdArgs := []string{"--name=etcd",
+	etcdArgs := []string{"--name=etcd1",
 		"--advertise-client-urls=http://0.0.0.0:2379",
 		"--listen-client-urls=http://0.0.0.0:2379",
+		"--initial-cluster=etcd1=http://0.0.0.0:2380",
+		"--initial-advertise-peer-urls=http://0.0.0.0:2380",
 		"--initial-cluster-state=new",
 		"--initial-cluster-token=new",
-		"--log-output=stdout",
-		"--data-dir=" + os.Getenv("ETCD_DATA_DIR")}
```

Comment:
Can you explain why this was removed?

Reply:
We are already providing `data directory` through config file. It's creating conflict if it's also provided by the cmd line arguments